docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-designing-a-case-proc.adoc

```diff
@@ -1,2 +1,99 @@
-[id='']
-= TODO
\ No newline at end of file
+[id='case-management-designing-a-case-proc']
+= Designing a case
+
+Cases are designed using the *Case definition* design editor in {CENTRAL}. The case design is the basis of case management and sets out the specific goals and tasks for each case. The design can be added to and modified dynamically during runtime.
+
+To demonstrate how to design a case, {PRODUCT} includes the IT Orders sample project in {CENTRAL}, which includes the following case definition:
+
+image::itorders-orderhardware-process.png[IT Orders case design]
+
+You can refer to and interact with this case definition by importing the sample project in {CENTRAL}. 
+
+.Prerequisite 
```

Comment:
No bullet necessary when a single prereq. This is the pattern in our other docs anyway. Open to counter argument if you'd like.